packages/apps/auth0-actions/src/change_password.ts

```diff
@@ -25,7 +25,8 @@ async function changePassword(
   if (recordNumber) {
     const patronRecordUpdate = await sierraClient.updatePassword(
       recordNumber,
-      newPassword
+      // Sierra ignores passwords after the 30th character
```

Comment:
Good find! Is this documented anywhere? Is it worth enforcing this in the Auth0 rules?

Reply:
I'm not sure about this, because I just copied it from the existing implementation, which was introduced here https://github.com/wellcomecollection/identity/pull/40. Now we're keeping passwords in Sierra it seems more important, but I'm not sure what to do about it as I don't know how/what we'd communicate to users about it...